backend/db/migrations/V1.0.24__add_reports_views.sql

```diff
@@ -0,0 +1,103 @@
+SET search_path TO pay_transparency;
+
+
```

Comment:
Can you also add a comment here with a link on how to setup views for prisma? I think if we ever use views in the future, this would probably be the first place someone might check for more information to see how it was done before. Doesn't have to be long. Can even reference your other comment if you want.